config/sync/field.field.node.stanford_page.su_basic_page_tags_filters.yml

```diff
@@ -0,0 +1,29 @@
+uuid: 34e6e613-d839-4ca2-b5b9-51bc59ce2e7e
+langcode: en
+status: true
+dependencies:
+  config:
+    - field.storage.node.su_basic_page_tags_filters
+    - node.type.stanford_page
+    - taxonomy.vocabulary.basic_page_filters
+id: node.stanford_page.su_basic_page_tags_filters
+field_name: su_basic_page_tags_filters
+entity_type: node
+bundle: stanford_page
+label: 'Basic Page Tags/Filters'
+description: 'These will be used as filters for the future Filtered Lists Paragraph for Basic Page. They do not display on the Basic Page or Basic Page Teasers.'
```

Comment:
We now need to update help text since when we release this, now the lists will be created! Sorry about that. I ran out of time to review for this release.
"Tag this page with categories for use with the Filtered List Paragraph. Tags do not display on the Basic Page or Basic Page Teasers. Learn more (and then link to https://sitesuserguide.stanford.edu/build/paragraphs/filtered-lists-paragraph.)